src/components/table/table-cell/table-cell.component.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StyledTableCell from './table-cell.style';
+import { validProps } from '../../../utils/ether';
+import tagComponent from '../../../utils/helpers/tags';
+import OptionsHelper from '../../../utils/helpers/options-helper';
+
+/**
+ * A TableCell widget.
+ *
+ * == How to use a TableCell in a component:
+ *
+ * See documentation for Table component.
+ *
+ * You can set a property of 'align' which should be a string. This will
+ * align the content to either "left", "center" or "right".
+ *
+ * You can set a property of 'action' which should be a boolean. This will
+ * set styling options for the cell used for action such as delete.
+ *
+ * @class TableCell
```

Comment:
Can remove these `@` descriptors I think. There's a few more throughout this file.